libpod/version.go

```diff
@@ -0,0 +1,50 @@
+package libpod
+
+import (
+	"runtime"
+	"strconv"
+
+	"github.com/projectatomic/libpod/cmd/podman/ioprojectatomicpodman"
+	podmanVersion "github.com/projectatomic/libpod/version"
+)
+
+// Overwritten at build time
+var (
+	// GitCommit is the commit that the binary is being built from.
+	// It will be populated by the Makefile.
+	GitCommit string
+	// BuildInfo is the time at which the binary was built
+	// It will be populated by the Makefile.
+	BuildInfo string
+)
+
+//VersionOutput is an output struct for varlink
+type VersionOutput struct {
+	ioprojectatomicpodman.VarlinkInterface
```

Comment:
Do we have to include this here? Are other structs going to need it?

Reply:
yes, all returned structs

Reply:
This means we're going to be throwing them in the DB. I'm not sure I like that.